firebase_admin/_user_mgt.py

```diff
@@ -26,6 +26,7 @@
 
 MAX_LIST_USERS_RESULTS = 1000
 MAX_IMPORT_USERS_SIZE = 1000
+B64_REDACTED = base64.b64encode("REDACTED")
```

Comment:
Add `b'REDACTED'` to fix the test failures on Python3

Reply:
Oops. (Fixed; thanks.)